src/Tabs/Tab-styled.js

```diff
@@ -153,18 +153,18 @@ const StyledTabTitle = styled(CalciteA)`
       ${props =>
         props.translucent &&
         css`
-          background-color: ${props => props.theme.palette.transparentWhite};
+          background-color: transparent;
           background-image: none;
           border: none;
-          border-top: 2px solid ${props => props.theme.palette.transparentWhite};
-          color: ${props => props.theme.palette.offBlack};
-          margin-right: 2px;
-          margin-bottom: 3px;
+          border-bottom: 3px solid transparent;
+          color: ${props.theme.palette.darkerGray};
+          margin-right: ${unitCalc(props.theme.baseline, 2, '/')};
           transition: none;
```

Comment:
How about we also update to match the transition for the updated calcite-components design? Looks like it'd be `transition: all 0.15s ease-in-out 0s;`